tests/lib/__init__.py

```diff
@@ -44,11 +44,11 @@ def urllib_read(query_url):
 			return '127.0.0.1'
 		elif query_url.startswith('http://ipv4.icanhazip.com'):
 			return '2001:4801:7818:6:abc5:ba2c:ff10:275f'
-	elif query_url.startswith('http://freegeoip.net/json/'):
-		return '{"city": "Meppen", "region_code": "06", "region_name": "Niedersachsen", "areacode": "", "ip": "82.145.55.16", "zipcode": "49716", "longitude": 7.3167, "country_name": "Germany", "country_code": "DE", "metrocode": "", "latitude": 52.6833}'
+	elif query_url.startswith('http://geoip.nekudo.com/api/'):
+		return '{"city": "Meppen", "region_code": "06", "region_name": "Niedersachsen", "areacode": "", "ip": "82.145.55.16", "zipcode": "49716", "longitude": 7.3167, "country": {"name": "Germany", "code": "DE"}, "metrocode": "", "latitude": 52.6833}'
```

Comment:
I see `geoip.nekudo.com` returns something like `{"city":"Moscow","country"…`. Note: *no* spaces. Do *not* write this by hand, copy actual response from the service.